src/utils/prism-import.js

```diff
@@ -1,5 +1,3 @@
-import 'prismjs/components/prism-core';
```

Comment:
Why are we removing prism-core? This is used for the playground via react-live

Reply:
AFAIRemember it's removed since importing this overwrites the instance of `Prism` in `window`, removing all configurations made by `react-live`. It's imported by `react-live` as well, which is why I came to the conclusion that the import is no longer needed. (And it worked, AFAIRemember.)

Reply:
I'm guessing piggybacking off a separate library's dependency is not a good long term approach and we really need to solve the underlying prismjs version conflict problem discussed in detail in #546 first...

/cc @kenwheeler @kitten

Reply:
I agree but this was the solution I could find for #526. Maybe it would still be a valid approach to merge this and refactor this through #546?

Reply:
Are we completely confident relying on the load order of `react-live` and then piggy-backing here won't break? Including if `react-live` changes semver under the hood what _it_ does with `prism`? I'm guessing we can't make that guarantee...

Reply:
Basically, I don't want to open up _new_ bugs for a hack here when we have a larger overall issue that needs solving 😄

Reply:
Load order: I'm confident, there's no asynchronously happening.

Semver: we can never guarantee that changes in `react-live` don't break anything here. Currently it already *is* broken since no languages other than the default of `react-live` can be used. The issue here is with how `prism` works: it creates a global object and syntax settings are added to this. Re-importing `prism` will overwrite the global object, thus removing all prior settings. This also means: importing `prism-import.js` has no effect _at all_ the way it is now (without the PR).